modules/monitoring-creating-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -8,6 +8,13 @@
 
 You can create alerting rules for user-defined projects. Those alerting rules will trigger alerts based on the values of the chosen metrics.
 
+[NOTE]
+====
+* When you create an alerting rule, a project label is enforced on it even if a rule with the same name exists in another project.
+
+* Including the alert message and severity in your YAML configuration file can explain why the alert is triggered and the impact of the alert.
```

Comment:
This comment also applies to the other places where this sentence repeats:

The wording "Including...can explain" isn't sitting right with me. I think it's not quite specific enough - who is this being explained to? There also isn't mention of the YAML at this point so I'm not sure it is helpful to specify it yet.

Here's a suggestion to consider:

```suggestion
* To help users understand the impact and cause of the alert, ensure that your alerting rule contains an alert message and severity value.
```

Reply:
Hello @ousleyp I love your suggestion, thank you for that! It is implemented now :)